### ISSUE TITLE
Special treatment for follower XP improvement request

### ISSUE THREAD

**Issue by irandamay** [TLDRMissions/TLDRMissions#294]
TLDR version: 
If you have fewer followers that need to level up than the minimum set for the special treatment, then send them on the bonus missions.

**Slightly longer version**

I noticed a few days ago that TLDR had stopped sending my leveling guys out on bonus xp missions even though bonus xp is set as my top priority and I have the special treatment setting turned on.  It was just sending my guys out individually onto lower priority missions for much less xp.  Finally this morning, I decided I was going to see if there were any other bug reports about this and possibly file one after seeing it try to send a guy on a 900 xp mission rather than the 1200 + 2000 bonus one that was available.

It turns out that I had not realized there are new options for the special treatment setting for minimum followers, and it seems to have defaulted to a minimum of 4 followers.  However, I only have 3 followers left that I am trying to level up, so it was seemingly never choosing the bonus xp missions anymore because I don't have 4 followers that still need to level.

Once I lowered the minimum to 1 follower and recalculated, it was happy to pick the mission worth 3200 xp as the first mission.

It would be nice if possible if it took how many you have left that need to level at all into account for this setting.

**Comment by teelolws** [comment]
That setting was added because other players told me "Its a waste to send only one or two followers on this mission". But it seems like you have changed it to minimum of 1 follower, which will do what you want now, no?
